Gruntfile.js

```diff
@@ -342,19 +348,165 @@ module.exports = function (grunt) {
 							]
 						},
 						{
-							config: 'bump.version',
+							// if custom bump is used with a specific version, see dorelease task
+							config: 'release.buildSemVerType',
 							type: 'input',
 							message: 'What specific version would you like',
 							when: function (answers) {
-								return answers['bump.increment'] === 'custom';
+								return answers['release.buildSemVerType'] === 'custom';
 							},
 							validate: function (value) {
 								var valid = semver.valid(value);
 								return valid || 'Must be a valid semver, such as 1.2.3-rc1. See http://semver.org/ for more details.';
 							}
+						},
+						{
+							config: 'release.remoteRepository',
+							default : '<%= release.remoteRepository %>',
+							type: 'input',
+							message: function() {
+								return 'What repository would like to base your local release branch from?';
+							}
+						},
+						{
+							// Assumption is made that you are releasing the code within a "release branch" currently 
+							// on the upstream remote repo. This branch will be tracked locally and be used to run 
+							// the build process in. It will be named release_{BUILD_VERSION}_{MMSS} (that is, it will
+							// use the version specified earlier and a "mini-timestamp" of the current hour and minute).
+							config: 'release.remoteBaseBranch',
+							type: 'input',
+							default : '<%= release.remoteBaseBranch %>',
+							message: function() {
+								return 'What remote branch from ' + grunt.config('release.remoteRepository') + 
+								' would like to build your release based upon?';
```

Comment:
I'm [old fashioned](http://www.quora.com/What-is-the-difference-between-on-and-upon), I guess. 'On' is probably easier to understand in a publicly accessible repository for non-native English speakers.